### DISCUSSION TITLE
How to add inner element class selector in CSS style manager

### DISCUSSION THREAD

**Discussion by msayanece** [GrapesJS/grapesjs#3964] [Q&A]
I need a way to add a class/id to the inner element of the selected component using style manager like this...

`".el-class .inner-class"`

Currently space is not allowed inside the class selector in style manager. Is there any way to do that?

**Comment by artf** [comment]
Kind of... you can try `editor.StyleManager.setTarget('.el-class .inner-class')` but this is an experimental method (this is why is undocumented). 
What I can tell is that I'm working on the StyleManager refactoring and this kind of feature is on my list.
